### PR TITLE
Adding bank poalim

### PR DESCRIPTION
**Pull request by xFreed0m**

### CODE REVIEW

File: content/sponsors/bankpoalim.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+key: bankpoalim
+title: Bank Poalim
+category: gold
+website: ''
+socials: []
+---
+
+Check out Bank Poalim @
```
Comment (omercnet) [inline content/sponsors/bankpoalim.md:9]: @xFreed0m need link

**Review by omercnet — CHANGES_REQUESTED** [review]
rebase..

File: content/sponsors/bankpoalim.md (hunk 1 above)
Comment (xFreed0m) [reply content/sponsors/bankpoalim.md:9]: we don't have one currently

Comment (omercnet) [reply content/sponsors/bankpoalim.md:9]: so let's remove the text
```suggestion

```

**Review by omercnet — CHANGES_REQUESTED** [review]

**Review by omercnet — APPROVED** [review]

**Pull request merged**